k2/csrc/fsa_algo.cc

```diff
@@ -52,6 +52,34 @@ inline int32_t InsertIntersectionState(
   return result.first->second;
 }
 
+/**
+   A TraceBack() function used in RmEpsilonPrunedLogSum.  It finds derivative
```

Comment:
The function name is `RmEpsilon **s** PrunedLogSum`.

Reply:
Oh, yes, updated, thanks!